src/Conf.ml

```diff
@@ -975,12 +975,21 @@ module Formatting = struct
 
   let indicate_multiline_delimiters =
     let doc =
-      "Print a space inside a delimiter to indicate that its matching \
-       delimiter is on a different line."
+      "How to indicate that two matching delimiters live on different lines."
     in
     let names = ["indicate-multiline-delimiters"] in
-    let default = true in
-    C.flag ~names ~default ~doc ~section
+    let all =
+      [ ( "space"
+        , `Space
+        , "$(b,space) prints a space inside the delimiter to indicate the \
+           matching one is on a different line" )
```

Comment:
Could you add dots at the end of the sentences ? Otherwise the generated doc is hard to read.

Reply:
Fixed.